examples/simulation/Tutorial1/src/FairSimConfig.cxx

```diff
@@ -1,32 +1,27 @@
-/********************************************************************************
- * Copyright (C) 2014-2023 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH  *
- *                                                                              *
- *              This software is distributed under the terms of the             *
- *              GNU Lesser General Public Licence (LGPL) version 3,             *
- *                  copied verbatim in the file "LICENSE"                       *
- ********************************************************************************/
+/*********************************************************************************
+ *  Copyright (C) 2014-2023 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH  *
+ *                                                                               *
+ *              This software is distributed under the terms of the              *
+ *              GNU Lesser General Public Licence (LGPL) version 3,              *
+ *                  copied verbatim in the file "LICENSE"                        *
+ *********************************************************************************/
 #include "FairSimConfig.h"
 
 #include "FairLogger.h"
 
+#include <boost/program_options.hpp>
+namespace po = boost::program_options;
+
 #include <iostream>
 #include <vector>
 
 using std::string;
 using std::vector;
 
-FairSimConfig::FairSimConfig()
-    : fDescription("Options")
-    , fHelp(false)
-    , fnEvents(1)
-    , fEngine("TGeant3")
-    , fMultiThreaded(false)
-    , fOutputFile("sim.root")
-    , fParameterFile("par.root")
-    , fRandomSeed(0)
-{
+auto makeDescription() {
+    po::options_description description;
```

Comment:
1. Please make it static, so that it does not pollute the global namespace
2. The original ctor of fDescription was called with `"Options"`. Do we need this?

```suggestion
static auto makeDescription() {
    po::options_description description{"Options"};
```

Reply:
FYI: The `"Options"` argument gives the help output a title line.

Reply:
Fixed